server/src/com/cloud/api/dispatch/ParamProcessWorker.java

```diff
@@ -145,7 +146,7 @@ private void validateField(final Object paramObj, final Parameter annotation) th
 
     @SuppressWarnings({"unchecked", "rawtypes"})
     public void processParameters(final BaseCmd cmd, final Map params) {
-        final Map<Object, AccessType> entitiesToAccess = new HashMap<Object, AccessType>();
+        final Multimap<Object, AccessType> entitiesToAccess = ArrayListMultimap.create();
```

Comment:
Any particular reason, we changed it to multimap?

Reply:
This is to check both account and domain vnet tables for a given entity id